src/App.js

```diff
@@ -31,34 +34,44 @@ const StyledResultsContainer = styled.div`
 
 function App() {
   //adding pieces of state with the currentPrice of our currency, our Reference selection, our BetBehaviour and the NextPrice
+  const [currency, setCurrency] = useState('BTC');
   const [currentPrice, setCurrentPrice] = useState(null);
   const [referencePrice, setReferencePrice] = useState(null);
   const [betType, setBetType] = useState(null);
   const [betPrice, setBetPrice] = useState(null);
   const [showResults, setShowResults] = useState(false);
   const [message, setMessage] = useState(null);
+  const [credits, setCredits] = useState(3);
 
   // instantiating our currency formatter
   let formattedCurrency = new Intl.NumberFormat('en-US', {
     style: 'currency',
     currency: 'USD',
   }).format(currentPrice);
 
-  // function to setReferencePrice when button onClick
+  // function to setReferencePrice when Button onClick
   const selectReference = () => {
     setReferencePrice(formattedCurrency);
     console.log('reference: ' + formattedCurrency);
   };
 
+  // function to select t(he currency
+  const selectCurrency = (e) => {
+    console.log(e.target.value);
+    setCurrency(e.target.value);
+  };
+
   // once we check the price we activate our modal to show the results of our bet
   const checkPrice = () => {
     setShowResults(true);
     // We set the bet price in order to pass it to our modal and be able to display it at part of summary
     setBetPrice(formattedCurrency);
     if (referencePrice > formattedCurrency && betType === UP_TYPE) {
       setMessage('You loose!');
+      setCredits(credits - 1);
     } else if (referencePrice < formattedCurrency && betType === DOWN_TYPE) {
       setMessage('You loose!');
+      setCredits(credits - 1);
     } else if (referencePrice === formattedCurrency) {
       setMessage("It's a tie, try again");
     } else {
```

Comment:
You win! 
Maybe we should increment the credits they have? 

A fun feature we could implement is, say you get to 10 credits - then we show confetti or some animation. 
"Woohoo! You're the smartest person alive!"

Reply:
yeah! let's do that :)